### PR TITLE
Add a couple clarifications to Custom Images contract.

### PR DESCRIPTION
**Pull request by chekunkov**
Provide more details on how crawlers are started.

### CODE REVIEW

File: docs/custom-images-contract.rst [hunk 1]
```diff
@@ -28,6 +29,11 @@ Contract statements
 
     docker run myscrapyimage start-crawl
 
+   Crawler will be started as unpriviledged user ``nobody`` in a writable directory ``/scrapinghub``.
+   ``HOME`` environment variable will be set to ``/scrapinghub`` as well. Beware that this directory
+   is added dynamically when the jobs starts. If custom image contains this directory -
+   it'll be erased on job start - so
```
Comment (vshlapakov) [inline docs/custom-images-contract.rst]: Is it intended to keep the sentence unfinished?

File: docs/custom-images-contract.rst [hunk 2]
```diff
@@ -28,6 +29,11 @@ Contract statements
 
     docker run myscrapyimage start-crawl
 
+   Crawler will be started as unpriviledged user ``nobody`` in a writable directory ``/scrapinghub``.
+   ``HOME`` environment variable will be set to ``/scrapinghub`` as well. Beware that this directory
+   is added dynamically when the jobs starts. If custom image contains this directory -
```
Comment (vshlapakov) [inline docs/custom-images-contract.rst]: `when the job starts`

File: docs/custom-images-contract.rst [hunk 3]
```diff
@@ -28,6 +29,11 @@ Contract statements
 
     docker run myscrapyimage start-crawl
 
+   Crawler will be started as unpriviledged user ``nobody`` in a writable directory ``/scrapinghub``.
```
Comment (vshlapakov) [inline docs/custom-images-contract.rst]: Probably,  `started *by* unprivileged user`?

File: docs/custom-images-contract.rst (hunk 1 above)
Comment (chekunkov) [inline docs/custom-images-contract.rst]: lol 🤦‍♂️

**Comment by chekunkov** [comment]
@vshlapakov thanks for the review, I fixed the issues

**Pull request merged**